lib/ldu_email_importer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class LDUEmailImporter
+  def self.import(filename)
+    CSV.read(filename, headers: true).each do |row|
+      code = row[0]&.strip
+      address = row[1]&.strip
+
+      next if address.blank?
+
+      LocalDivisionalUnit.where(code: code).update_all(email_address: address)
```

Comment:
update_all isn't a good method to use except in a migration - it bypasses validations. Just update! is probably fine here.